docs/pagination.md

```diff
@@ -0,0 +1,71 @@
+# Pagination
+
+Pagination is supported using the metadata support that is built into Ember Data. The
+metadata from Django REST Framework paginated list views is updated on every request
+to sever.
+
+
+## Retrieving the Metadata
+
+To get a page of records simply run a find request with the `page` query param:
+
+```js
+var result = this.store.find("post", {
+  page: 1
+});
+```
+
+It's possible to override the server-side page size by also including the `page_size`
```

Comment:
Users may just set the `PAGINATE_BY` setting in DRF and forego the `PAGINATE_BY_PARAM` setting altogether—which is fine, and simpler.  So this should be optional and off by default on our end as well.  

Can we make this customizable, so that the user can specify the param name, if any, just like they can in DRF?